### ISSUE TITLE
Reports > To Do

### ISSUE THREAD

**Issue by shaedrich** [foundeo/cfdocs#519]
Under functions the last dropdown list item in navigation is "Todo" which links to [/reports/todo.cfm](https://cfdocs.org/reports/todo.cfm).

There "Todo" is written as one word; the "To Do" item in root level of the navigation is written as two. That's inconsistent.

Tags does not have an own "To Do" page.

[/reports/todo.cfm](https://cfdocs.org/reports/todo.cfm) seems to be the top level todo list because [/reports/missing-descriptions.cfm](https://cfdocs.org/reports/missing-descriptions.cfm), [/reports/missing-examples.cfm](https://cfdocs.org/reports/missing-examples.cfm) and [/reports/missing-related.cfm](https://cfdocs.org/reports/missing-descriptions.cfm) have their names included in url.

In https://github.com/foundeo/cfdocs/issues/516#issuecomment-325007705 you told me that you distinguish between between CFML as all and cfscript as a subset of it. All three todo lists (["Missing Descriptions"](https://cfdocs.org/reports/missing-descriptions.cfm), ["Missing Examples"](https://cfdocs.org/reports/missing-examples.cfm) and ["Missing Related Links"](https://cfdocs.org/reports/missing-descriptions.cfm)) have a clear two column grid separation of tags and functions.

So you have two kinds of todos:
* "Missing x" in both tags and functions
* Missing tag or function itself

Should this be depicted in "To Do" dropdown navigation menu?

Should there perhaps be a "To Do" overview page which may include
* Boxes or cards linking to the todo subpages
* Statistics like "571 functions documented", "889 tags and functions in total", "87.29% descriptions complete"
* Specific examples how to solve the todos

**Comment by pfreitag** [comment]
Yeah, this is a bit patchy because it has changed over time and things have gotten added in by various people.  I think ultimately it would make more sense to remove the Todo Menu, and just have a _Contribute_ menu item which has an overview of how to contribute, and then sub links to the areas that currently need work.

**Comment by shaedrich** [comment]
Good idea :+1:

**Comment by shaedrich** [comment]
Should [CONTRIBUTING.md](/foundeo/cfdocs/blob/master/CONTRIBUTING.md) loaded as  first item named "How to" in that menu?

**Comment by shawnoden** [comment]
Love it when I dig a bit deeper into a question I have and find that I not only asked pretty much the same question 5 years ago, but got an answer and submitted a PR for it. #421

I can do this again for 2021, if needed/wanted, but it also goes to my other question of should the To Do page that lists missing functions be added back in some way?
